request/graphql/schema/types/types.go

```diff
@@ -24,6 +24,10 @@ const (
 	ExplainArgExecute  string = "execute"
 	ExplainArgDebug    string = "debug"
 
+	PolicySchemaDirectiveLabel        = "policy"
```

Comment:
todo: Please define the directive bellow and then add it to `defaultDirectivesType()` in `defradb/request/graphql/schema/manager.go`.

Reply:
Hey I think I did that in a merge conflict I had today, LMK if something is still missing.